krun/platform.py

```diff
@@ -294,6 +299,20 @@ def _sanity_check_user_change(self):
         util.spawn_sanity_check(self, ep, vd, "UNIX user change",
                                 force_dir=PLATFORM_SANITY_CHECK_DIR)
 
+    def sync_disks(self):
+        """Force pending I/O to physical disks"""
+
+        debug("sync disks")
+        rc = subprocess.call("/bin/sync")
+        if rc != 0:
+            fatal("sync failed")
+
+        # The OpenBSD manual says: "sync() [the system call] may return before
+        # the buffers are completely flushed.", and the sync command is merely
+        # a thin wrapper around the syscall. We wait a second (should be more
+        # than enough time for disks to sync).
```

Comment:
Does the `sync` call return immediately? At least on OpenBSD, the `sync` shell command seems to wait until the syncing is complete (I don't know how).